code/game/objects/structures/fluff.dm

```diff
@@ -22,6 +22,11 @@
 		return
 	..()
 
+/obj/structure/fluff/minibar
```

Comment:
this is supposed to allow you to place objects on it, which is probably why it was a rack before.